src/jmcomic/jm_plugin.py

```
@@ -799,6 +799,84 @@ def ensure_make_pdf_dir(pdf_dir: str):
+
+    def invoke(self,
+               photo: JmPhotoDetail = None,
+               album: JmAlbumDetail = None,
+               downloader=None,
+               img_dir=None,
+               filename_rule='Pid',
+               delete_original_file=False,
+               **kwargs,
+               ):
+        if photo is None and album is None:
+            jm_log('wrong_usage', 'long_img必须运行在after_photo或after_album时')
```

Ruff

`jm_log` may be undefined, or defined from star imports

(F405)